src/components/ImportJobs/ImportJobs.js

```diff
@@ -137,11 +140,23 @@ class ImportJobsComponent extends Component {
     }
 
     if (redirect) {
-      return <Redirect to={`${path}/job-profile`} />;
+      return (
+        <Redirect
+          to={{
+            pathname: `${path}/job-profile`,
+            state: { files },
+          }}
+        />
+      );
     }
 
     if (!isEmpty(uploadDefinition)) {
-      return <ReturnToAssignJobs prohibitFilesUploading={prohibitFilesUploading} />;
+      return (
+        <ReturnToAssignJobs
+          prohibitFilesUploading={prohibitFilesUploading}
+          onResume={() => this.redirectToJobProfilePage()}
```

Comment:
```suggestion
          onResume={this.redirectToJobProfilePage}
```

Reply:
That will not work correctly because `this.redirectToJobProfilePage` will take event object as an argument and the first argument must be an array of files or undefined